draft-ietf-acme-acme.md

```diff
@@ -322,12 +329,18 @@ The use of ACME for other identifiers will require further specification in orde
 to describe how these identifiers are encoded in the protocol and what types of
 validation challenges the server might require.
 
-# Character Encoding
+# Encodings
 
 All requests and responses sent via HTTP by ACME clients, ACME servers, and
 validation servers as well as any inputs for digest computations MUST be encoded
 using the UTF-8 {{!RFC3629}} character set.
 
+Some JSON structures used by ACME include date time values encoded
```

Comment:
s/date time/date-time/ ?
But it seems you say "date format" elsewhere.  Just do the right thing. :)